oliverli-gubilgut/hello/hello.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+const spanish = "Spanish"
+const french = "French"
+const forBetterHelloEnglish = "Hello, "
+const forBetterHelloSpanish = "Hola, "
+const forBetterHelloFrench = "Bonjour, "
+
+func Hello(name string, language string) string {
+	if name == "" {
+		name = "Asgardia"
+	}
+
+	return greetingPrefix(language) + name
+}
+
+func greetingPrefix(language string) (prefixOutput string) {
```

Comment:
personally I would prefer to stick to plain string as return value and declare the return variable inside the body and return it.

Reply:
hahah, i was waiting this from someone, great catch @dariuszparys ! :) i also stopped for a sec to think about this part while coding since strings are immutable and returning a value is a better implementation in most cases.. 

however the below part in the exercise distracted me (i found this very interesting and wanted to use "empty return".. so i kept it.. 
> It will be assigned the "zero" value. This depends on the type, for example ints are 0 and for strings it is "".
> You can return whatever it's set to by just calling return rather than return prefix.

so if you would be comfortable about it, i want to keep it since i still find this interesting, it seems harmless for this case and want to have it as the example of "empty return" as future syntax references.. if not, still we are cool, let me know it and i will change it ;)

Reply:
Of course you can keep it, my comments are mostly suggestions and I'm also keen to learn if this is maybe something you do in go